docs/docs/api.md

```diff
@@ -46,7 +48,7 @@ GET requests with parameters should have the parameters in the query string. POS
 API responses should always either have a body with a JSON result object (if the response was a 200)
 or with a JSON object representing an error (if the response was a 4xx or 5xx).
 
-All times are UNIX timestamps in milliseconds since epoch.
+All times are UNIX timestamps in milliseconds since epoch. (Javascript timestamps.)
```

Comment:
There's no such thing as a "Javascript timestamp."

Reply:
I don't think the terminology is particularly important, so if you don't want that in there, I won't object. But "Javascript timestamp" is meaningful, since it's distinguishable from Unix time by the fact that it's denoted in miliseconds. Unix time is in seconds.